apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.ts

```diff
@@ -206,6 +206,26 @@ router.post(
       } catch {
         res.redirect(res.locals.urlPrefix + '/edit_error/' + serverJob.jobSequenceId);
       }
+    } else if (req.body.__action === 'sharing_set_add') {
```

Comment:
When you merge master, you'll see that this endpoint is completely gone now, sharing info is specified only in the JSON. So you need to do the check to make sure draft questions aren't shared somewhere around here instead:

https://github.com/PrairieLearn/PrairieLearn/blob/93bfa942f579247242d75cd2179be150ddc547d2/apps/prairielearn/src/sync/sharing.ts#L105-L136